djhtml/modes.py

```diff
@@ -513,17 +523,14 @@ class InsideHTMLTag(DjTXT):
 
     RAW_TOKENS = DjTXT.RAW_TOKENS + [r"/?>", r"[^ ='\">/\n]+=", r'"', r"'"]
 
-    def __init__(self, tagname, line, return_mode):
+    def __init__(self, tagname, line, return_mode, absolute, offsets):
```

Comment:
I know, this signature isn't pretty. DjHTML 4 will use required kwargs and type annotations, I promise!